.github/workflows/test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         run: npm install
 
       - name: Run Tests
-        run: npm run test:unit
+        run: node --test --test-reporter=@reporters/github --test-reporter-destination=stdout --test-reporter=spec --test-reporter-destination=stdout --loader=tsx ./tests/e2e/index.test.ts
```

Comment:
we can add it directly in `package.json` - `@reporters/github` is a noop when runing outside of github actions